source/features/sticky-discussion-sidebar.tsx

```diff
@@ -4,7 +4,7 @@ import debounce from 'debounce-fn';
 import features from '../libs/features';
 
 function updateStickiness(): void {
-	const sidebar = select<HTMLElement>('.discussion-sidebar')!;
+	const sidebar = select<HTMLElement>('#partial-discussion-sidebar')!.parentElement!;
```

Comment:
Same here

Reply:
Again, that's what I thought at first too, unfortunately no,
as it leads to this issue again:
https://github.com/sindresorhus/refined-github/pull/2294#issuecomment-517205362

![nS24XMIXGj](https://user-images.githubusercontent.com/10238474/62282128-af6fb980-b457-11e9-8b18-c29687b88da1.gif)